ts/features/services/home/store/reducers/__tests__/store.test.ts

```diff
@@ -0,0 +1,334 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Action, createStore } from "redux";
+import { InstitutionsResource } from "../../../../../../../definitions/services/InstitutionsResource";
+import { OrganizationFiscalCode } from "../../../../../../../definitions/services/OrganizationFiscalCode";
+import { applicationChangeState } from "../../../../../../store/actions/application";
+import { appReducer } from "../../../../../../store/reducers";
+import { NetworkError } from "../../../../../../utils/errors";
+import { paginatedInstitutionsGet } from "../../actions";
+import {
+  isErrorPaginatedInstitutionsSelector,
+  isLoadingPaginatedInstitutionsSelector,
+  isUpdatingPaginatedInstitutionsSelector,
+  paginatedInstitutionsSelector
+} from "..";
+import { GlobalState } from "../../../../../../store/reducers/types";
+
+const MOCK_INSTITUTIONS: InstitutionsResource["institutions"] = [
+  {
+    fiscal_code: "FRLFNC82A04D969A" as OrganizationFiscalCode,
+    id: "1",
+    name: "Fornitori"
```

Comment:
From a purely functional point of view, I suggest inserting different names for different institutions _(eg.: Institution 1, Institution 2, ...)_

Reply:
Done it https://github.com/pagopa/io-app/pull/5701/commits/849ea27a445a3fe41f96c9dbcf5ae0590d95956e